moveit_ros/hybrid_planning/test/test_basic_integration.cpp

```diff
@@ -92,7 +89,7 @@ class HybridPlanningFixture : public ::testing::Test
     tf_buffer_ = std::make_shared<tf2_ros::Buffer>(node_->get_clock());
 
     planning_scene_monitor_ = std::make_shared<planning_scene_monitor::PlanningSceneMonitor>(
-        node_, "robot_description", tf_buffer_, "planning_scene_monitor");
+        node_, "robot_description" /*, tf_buffer_*/, "planning_scene_monitor");
```

Comment:
I'm confused, you don't need to pass this parameter.  Why are you just commenting it out?

Reply:
Still in the middle of changes. Hang tight.